Makefile.js

```diff
@@ -58,9 +58,13 @@ var NODE = "node ", // intentional extra space
     JS_FILES = find("lib/").filter(fileType("js")).join(" "),
     JSON_FILES = find("conf/").filter(fileType("json")).join(" ") + " .eslintrc",
     MARKDOWN_FILES_ARRAY = find("docs/").concat(ls(".")).filter(fileType("md")),
-    TEST_FILES = find("tests/lib/").filter(fileType("js")).join(" ");
+    TEST_FILES = find("tests/lib/").filter(fileType("js")).join(" "),
     /* eslint-enable no-use-before-define */
 
+    // Regex
+    TAG_REGEX = /^[Fix\:|Update\:|Breaking\:|Docs\:|Build\:|New\:|Upgrade\:]/,
+    ISSUE_REGEX = /[\(fixes|\(refs] #\d+\)$/;
```

Comment:
That's a good idea to allow for multiple fixes/refs, but I think it should be looser.  For example, the pattern above would fail for https://github.com/eslint/eslint/commit/c6cd0cdc06becce66a3bc58f5e6c6e017b7b2d3c, (`Fix: workaround for leading and trailing comments in padded-block (fixes #2336 and fixes #2788)`)

I'd like to propose instead: `/\((?:fixes|refs) .*#\d+\)$/`

It's pretty loose, but I think as long as there is evidence of someone trying to use the format, they've probably done it correctly.  If there is a nuanced problem with their fixes/refs statement, it can also be caught manually.